lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -121,6 +121,17 @@ function dosomething_mbp_update_7006(&$sandbox) {
 }
 
 /**
+ * Updates user_register production to:
+ * - include imageProcessingQueue
+ * - remove userCampaignActivityQueue
+ * from transactional_campaign_reportback production.
+ */
+function dosomething_mbp_update_7007(&$sandbox) {
+  db_drop_table('message_broker_producer_productions');
+  dosomething_mbp_install_productions();
```

Comment:
Dropping the table seems a little... dangerous. Is there a chance this will mess things up in prod when this script is running, and the table doesn't exist?

An alternative approach could be to just delete the record and insert one (instead of killing the entire table)